README.md

```diff
@@ -213,5 +213,5 @@ functions-framework --help
 
 ## For more information
 
-See the examples directory for additional examples, and consult the gem
-documentation on rubydoc.info for full reference documentation.
+See the examples directory for additional examples, and consult
+https://rubydoc.info/gems/functions_framework for full reference documentation.
```

Comment:
Does this link really have full reference documentation? I couldn't find useful information on the linked document.

Reply:
The reference docs aren't there yet because we haven't done a full release of the library yet. That site will rebuild automatically with the released reference docs when the library actually gets pushed to the package manager.